src/App.js

```diff
@@ -1,31 +1,9 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
 import "./App.css";
-import { Input, Button } from "antd";
-import "antd/dist/antd.css";
-import firebase from "firebase/app";
-import "firebase/database";
+import database from "./database.js";
+import GamePage from "./game_components/GamePage.js";
```

Comment:
standard naming is just a `/components/` folder, also you should pretty much never use snake_case